src/it/scala/deduplication/TestUtils.scala

```diff
@@ -0,0 +1,117 @@
+package com.ovoenergy.comms.deduplication
```

Comment:
We already have a trait in domain called "DynamoDbFixture" which enables the creation of tables, waiting for creation, deletion... This is something @ducanhkhu-ovo wrote. Can you not use that instead ?

Reply:
This library is OSS and does not depend on the domain library. It also has a lot of other things written from scratch to keep it self-container or only depend on public available libraries.

Reply:
Fair enough, did not realise this was OSS